src/common.c

```diff
@@ -557,3 +559,17 @@ fd_set_close_on_exec(int fd)
 }
 #endif
 
+bool
+isV6AddrSet(struct in6_addr *ip6)
```

Comment:
follow naming convention. also, can this replaced by using the comparison function vs a constant?